tools/piputil.py

```diff
@@ -59,16 +59,16 @@ def pip_install_dir(path, requirements_filepath):
 
     subprocess.check_call(pip_cmd)
 
-def create_requirementsfile(filename, req_txt=None):
-    if not req_txt:
-        req_txt = ('''
+def create_requirementsfile(filename, req_text=None):
+    if not req_text:
```

Comment:
We should probably fail fast on invalid input, rather that silently converting it, but not a big deal.